openshift_images/image-configuration-hcp.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="image-configuration-hcp"]
+= Image configuration resources (HCP)
+include::_attributes/common-attributes.adoc[]
+:context: image-configuration-hcp
+
```

Comment:
https://docs.google.com/document/d/165Lf_4yfVR4nv8g55Qy0pr4fLaYYtlK5l5VLHXGhNSc/edit
Content can be placed in the file, needs "rosa" command example, etc.